src/metabase/util/malli/fn.clj

```diff
@@ -154,27 +153,16 @@
                                                   [{:keys [value message]}]
                                                   (str message ", got: " (pr-str value)))})
             details   (merge
-                        {:type      error-type
-                         :error     error
-                         :humanized humanized
-                         :schema    schema
-                         :value     value}
-                        error-context)]
-        (if (or config/is-dev?
-              config/is-test?)
-          ;; In dev and test, throw an exception.
-          (throw (ex-info (case error-type
-                            ::invalid-input  (i18n/tru "Invalid input: {0}" (pr-str humanized))
-                            ::invalid-output (i18n/tru "Invalid output: {0}" (pr-str humanized)))
-                          details))
-          ;; In prod, log a warning.
-          (log/warn
-            (case error-type
-              ::invalid-input  (i18n/tru "Invalid input - Please report this as an issue on Github: {0}"
-                                         (pr-str humanized))
-              ::invalid-output (i18n/tru "Invalid output - Please report this as an issue on Github: {0}"
-                                         (pr-str humanized)))
-            details))))))
+                       {:type      error-type
+                        :error     error
+                        :humanized humanized
+                        :schema    schema
+                        :value     value}
+                       error-context)]
+        (throw (ex-info (case error-type
+                          ::invalid-input  (i18n/tru "Invalid input: {0}" (pr-str humanized))
+                          ::invalid-output (i18n/tru "Invalid output: {0}" (pr-str humanized)))
```

Comment:
I think the Github callout was useful since it's a "our code is definitely wrong" thing and not just "something has gone wrong that may or may not be our problem; exercise judgment when deciding what to do "

Reply:
The github callout is dead code though right, since we only exercise these checks in dev and test mode

Reply:
See https://github.com/metabase/metabase/pull/43942/files#r1633515608

Reply:
I think we should make the github callout work instead of ripping it out, though.

Reply:
Actually, is that code dead? If I have a namespace with the `:instrument/always` metadata, then I think the github callout bit is reachable in prod, no?